handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -538,18 +548,14 @@ public void flush(ChannelHandlerContext ctx) throws Exception {
         }
         try {
             wrap(ctx, false);
-        } catch (Throwable cause) {
-            // Fail pending writes.
-            pendingUnencryptedWrites.removeAndFailAll(cause);
```

Comment:
I'm a bit more pessimistic I guess ... we are calling the JDK's SSL APIs which can be backed by a boat load of complexity ... can you make me more comfortable and explain why none of these APIs will throw an unchecked exception under any condition, or if they do that we will handle it gracefully in some other way.